src/widgets/search-box/search-box.js

```diff
@@ -243,54 +190,81 @@ searchBox({
  *   instantsearch.widgets.searchBox({
  *     container: '#q',
  *     placeholder: 'Search for products',
- *     autofocus: false,
- *     poweredBy: true,
- *     reset: true,
- *     loadingIndicator: false
  *   })
  * );
  */
 export default function searchBox({
   container,
   placeholder = '',
-  cssClasses = {},
-  poweredBy = false,
-  wrapInput = true,
-  autofocus = 'auto',
-  searchOnEnterKeyPressOnly = false,
-  reset = true,
-  magnifier = true,
-  loadingIndicator = false,
+  cssClasses: userCssClasses = {},
+  autofocus = false,
+  searchAsYouType = true,
+  showReset = true,
+  showSubmit = true,
+  showLoadingIndicator = true,
   queryHook,
+  templates,
 } = {}) {
   if (!container) {
     throw new Error(usage);
   }
 
   const containerNode = getContainerNode(container);
 
-  // Only possible values are 'auto', true and false
-  if (typeof autofocus !== 'boolean') {
-    autofocus = 'auto';
+  if (containerNode.tagName === 'INPUT') {
```

Comment:
Hello,
Why would you force user to not use an already existing input as a search input?

Reply:
This change was made to be more consistent across all widgets in all InstantSearch flavors. You probably want to use [`connectSearchBox`](https://www.algolia.com/doc/api-reference/widgets/searchbox/js/#connector) now.

Please [file an issue](https://github.com/algolia/instantsearch.js/issues/new/choose) if you have trouble migrating.